tests/Doctrine/Tests/ORM/Hydration/AbstractHydratorTest.php

```diff
@@ -14,21 +14,21 @@
 use Doctrine\ORM\Query\ResultSetMapping;
 use Doctrine\ORM\UnitOfWork;
 use Doctrine\Tests\OrmFunctionalTestCase;
-use PHPUnit_Framework_MockObject_MockObject;
+use PHPUnit\Framework\MockObject\MockObject;
```

Comment:
Like mentioned above, this `use` doesn't seem to be used in the code.

Reply:
It's L25, L28, L31